fdbclient/Knobs.h

```diff
@@ -160,7 +160,6 @@ class ClientKnobs : public Knobs {
 	double BACKUP_STATUS_DELAY;
 	double BACKUP_STATUS_JITTER;
 	double MIN_CLEANUP_SECONDS;
-	bool RESTORE_IGNORE_LOG_FILES;   // Default is false. When set to true, the log files will be ignored during the restore, which can produce inconsistent restored data.
```

Comment:
Removing this knob seems like too large of a behaviour change to put in a patch release, I think this knob could still be used as the default value in the above line:
```
wait(store(inconsistentSnapshotOnly, restore.inconsistentSnapshotOnly().getD(tr, false, false)));
```

Reply:
This is from #4704. Do you suggest we change the master branch as well?

Reply:
On master, it's okay to remove a knob, but this will go into a patch release, so I think we need to keep the knob only in this PR.

Reply:
Thanks, Done.